src/config/dark-sites.config

```diff
@@ -771,6 +774,7 @@ yandexdataschool.ru/$
 yasio.pl
 yohoho.io
 you.dj
+youtube.com
```

Comment:
Won't need to open youtube to know this one isn't dark either.

Reply:
Looks like Youtube US may only have themes and hasn't rolled it out internationally yet also. And this is with the dark mode add-on disabled.
<img width="1381" alt="image" src="https://user-images.githubusercontent.com/73370925/109157569-2a867800-7740-11eb-9ccf-3665ae9eda53.png">